statefun-examples/statefun-python-walkthrough-example/walkthrough.py

```diff
@@ -111,12 +115,29 @@ def state2(context, message):
     context.state('counter').pack(counter)
 
 
-@functions.bind("walkthrough/state_access_del")
+@functions.bind(
+    typename="walkthrough/state_access_del",
+    states=[StateSpec("counter")])
 def state3(context, message):
     # state can be deleted easily by using the del keyword.
     del context['counter']
 
 
+@functions.bind(
+    typename="walkthrough/missing_state",
+    states=[
+        StateSpec("counter"),
+        StateSpec("missing-state-1"),
+        StateSpec("missing-state-2", expiration=Expiration(10000)),
+        StateSpec("missing-state-3", expiration=Expiration(2000, expire_mode=Expiration.Mode.AFTER_WRITE))
```

Comment:
What do you think about simplifying a bit the expiration object?
having to type the `expire_mode` is a bit too much, what do you think of:
1) having separate Expiration classes for each mode.
```StateSpec("seen", expiration=AfterWrite(..))``
```StateSpec("age", expiration=AfterReadOrWrite(..)))```

2) We use Python`s `timedelta` in `send_after` what do you think of using the same thing here?
```StateSpec("age", expiration=AfterWrite(timedelta(days=3)))```

Reply:
Very good ideas! I didn't realize the `timedelta` existed, makes sense to use that as well.